src/test/fuzz/integer.cpp

```diff
@@ -65,11 +67,16 @@ void test_one_input(const std::vector<uint8_t>& buffer)
     (void)DecompressAmount(u64);
     (void)FormatISO8601Date(i64);
     (void)FormatISO8601DateTime(i64);
+    // FormatMoney(i) not defined when i == std::numeric_limits<int64_t>::min()
+    if (i64 != std::numeric_limits<int64_t>::min()) {
+        (void)FormatMoney(i64);
```

Comment:
Can test a roundtrip of `FormatMoney<->ParseMoney`?

Reply:
Very good idea! Thanks for the suggestion. Now added :)